assemblyline/datastore/helper.py

```diff
@@ -1232,6 +1237,10 @@ def save_or_freshen_file(
                     if self.file.update(sha256, operations, retry_on_conflict=8):
                         return
 
+            # Before we overwrite current_fileinfo make sure to OR the flags togeather
```

Comment:
```suggestion
            # Before we overwrite current_fileinfo make sure to OR the flags together
```
Can we explain the why in the comment, in case we question the is_supplementary OR'ing in the future?